Cartfile

```diff
@@ -0,0 +1,3 @@
+github "dhardiman/Fetch"
+github "owensd/json-swift" "master"
```

Comment:
Have you benchmarked it against `NSJSONSerialization`? As I understand it, it's doing all the parsing itself, so would be interesting to know what performance we might be leaving on the table by using a third party parser rather than just a swift-friendly wrapper...